src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -24,11 +24,25 @@ void shouldHandleNull() {
 
     @Test
     void shouldSearchStrings() {
+        //containsAnyIgnoreCase
         assertThat(containsAnyIgnoreCase("This is a string text.", "something")).isFalse();
 
         assertThat(containsAnyIgnoreCase("This is a string text.", "This")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "this")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
+
+        //containsAllIgnoreCase
+        assertThat(containsAllIgnoreCase("This is a string text.", "something")).isFalse();
```

Comment:
Um die Methode containsAllIgnoreCase zu testen, wären auch Tests sinnvoll die das Handling von null-Referenzen testen. Zum Beispiel so wie in der Methode shouldHandleNull in dieser Klasse.